astroquery/vizier/tests/test_vizier.py

```diff
@@ -159,6 +170,14 @@ def test_get_catalogs(patch_post):
     result = vizier.core.Vizier.get_catalogs('J/ApJ/706/83')
     assert isinstance(result, commons.TableList)
 
+def test_find_resource_then_get(patch_post):
+    reses = vizier.core.Vizier.find_catalogs('2009ApJ...706...83K')
+    res = reses.values()[0]
+    result = vizier.core.Vizier.get_catalogs(res)
+    assert isinstance(result, commons.TableList)
+    assert result.keys()[0].startswith(res.name)
```

Comment:
this is a python2-only idiom.  `tuple(results.keys())[0]` would work.  Ditto line 175 above